model/src/main/kotlin/ResolvedConfiguration.kt

```diff
@@ -36,6 +37,12 @@ import org.ossreviewtoolkit.utils.common.getDuplicates
  * TODO: Add further data.
  */
 data class ResolvedConfiguration(
+    /**
+     * All [PackageConfiguration]s that match the scan results in this [OrtResult].
```

Comment:
Nit: Why does this explicitly mention "scan result"? If a `PackageConfiguration` contains no `licenseFindingCurations` but `pathExcludes`, there's nothing scanner-specific about it.

Reply:
Package configurations are always matched against the provenance of the scan result they are applied to, e.g. path exclude paths could be different for source artifacts and repositories.